keyboards/scythe/keymaps/forties/keymap.c

```diff
@@ -0,0 +1,208 @@
+/* Copyright 2019 ENDO Katsuhiro <ka2hiro@kagizaraya.jp>
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include QMK_KEYBOARD_H
+
+// Defines the keycodes used by our macros in process_record_user
+#define _QWERTY 0
+#define _HAPPY 1
+#define _LOWER 2
+#define _RAISE 3
+#define _ADJUST 16
```

Comment:
Giving the layer numbers non-consecutive indexes increases the firmware size for no change in functionality.

```suggestion
#define _ADJUST 3
```

Also, changing these defines to an `enum` will make these definitions easier to manage.

```c
enum layers {
    _QWERTY,
    _HAPPY,
    _LOWER,
    _RAISE,
    _ADJUST,
};
```

Reply:
I changed layer defines to enum. Thanks.